nengo/utils/progress.py

```diff
@@ -168,16 +186,20 @@ def update(self, progress):
         """
         raise NotImplementedError()
 
+    def close(self):
+        """Closes the progress bar.
+
+        Indicates that not further updates will be made.
+        """
+        pass
+
 
 class NoProgressBar(ProgressBar):
```

Comment:
Given that there are a lot of Progress Bar implementations, would it be worth it to split them off into their own file for the sake of organization?

Reply:
I think it is better to keep them in a single file.